src/main/java/net/minecraftforge/items/pickup/ItemPickupReason.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.items.pickup;
+
+import com.google.common.collect.Maps;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+import net.minecraft.util.StringRepresentable;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Class representing a reason in which an ItemEntity is being picked up.
+ */
+public final class ItemPickupReason implements StringRepresentable, Comparable<ItemPickupReason>
```

Comment:
Why exactly is this comparable?

Reply:
No reason really, tbh don't know why I originally made it comparable, I will remove the Comparable interface.